pkg/rancher-desktop/integrations/windowsIntegrationManager.ts

```diff
@@ -294,24 +295,49 @@ export default class WindowsIntegrationManager implements IntegrationManager {
     }
   }
 
-  protected async syncDockerCompose() {
-    await Promise.all([
-      this.syncHostDockerCompose(),
-      ...(await this.supportedDistros).map(distro => this.syncDistroDockerCompose(distro.name)),
-    ]);
+  protected async syncDockerPlugins() {
+    const wslPluginNames = await this.getWslDockerCliPluginNames();
+    const hostPluginNames = await this.getHostDockerCliPluginNames();
+
+    const promises: Promise<void>[] = wslPluginNames.map(wslPluginName => this.syncHostDockerPlugin(wslPluginName) );
+
+    for (const hostPluginName of hostPluginNames) {
+      promises.push(
+        ...(await this.supportedDistros).map(distro => this.syncDistroDockerPlugin(distro.name, hostPluginName)),
+      );
+    }
+    await Promise.all(promises);
+  }
+
+  protected async getWslDockerCliPluginNames(): Promise<string[]> {
+    const resourcesBinDir = path.join(paths.resources, 'linux', 'bin');
+
+    return (await fs.promises.readdir(resourcesBinDir)).filter((name) => {
+      return name.startsWith('docker-') && !name.startsWith('docker-credential-');
+    });
+  }
+
+  protected async getHostDockerCliPluginNames(): Promise<string[]> {
+    const resourcesBinDir = path.join(paths.resources, os.platform(), 'bin');
```

Comment:
`os.platform()` better return `win32`… but yeah probably still good to keep it that way.